config/config_types.go

```diff
@@ -305,11 +306,12 @@ type FeaturesSupportedConfig struct {
 
 // FeaturesEnabledConfig stores the exchanges enabled features
 type FeaturesEnabledConfig struct {
-	AutoPairUpdates bool `json:"autoPairUpdates"`
-	Websocket       bool `json:"websocketAPI"`
-	SaveTradeData   bool `json:"saveTradeData"`
-	TradeFeed       bool `json:"tradeFeed"`
-	FillsFeed       bool `json:"fillsFeed"`
+	AutoPairUpdates bool                         `json:"autoPairUpdates"`
+	Websocket       bool                         `json:"websocketAPI"`
+	SaveTradeData   bool                         `json:"saveTradeData"`
+	TradeFeed       bool                         `json:"tradeFeed"`
+	FillsFeed       bool                         `json:"fillsFeed"`
+	Subscriptions   []*subscription.Subscription `json:"subscriptions,omitempty"`
```

Comment:
I considered the same.
1) But then I realised that everyone has to type it and talk about it a lot, and there really is only one type.
So it feels redundant to call it a websocket subscription.

2) Moreover, it comes down to concept vs implementation/tech.
Websocket is *how* we subscribe to the data, but the user concept is just that there's a subscription.
Sometimes we end up frontloading the tech into the concept naming, without adding any meaning.
To some people it might _seem_ novel or important that this is a websocket. But really it's not relevant, IMO.

With those two points considered, if you'd still like it renamed then I will 🙂

Reply:
Yeah, I'm cool with that 👍 👍 👍